summon/tasks/summon-conjur.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Download and unpack Summon-Conjur
+  unarchive:
+    src: https://github.com/cyberark/summon-conjur/releases/download/v{{summon_conjur.version}}/summon-conjur-{{summon_conjur.os}}.tar.gz
+    dest: /usr/local/bin
+    remote_src: yes
+    creates: /usr/local/bin/summon-conjur
```

Comment:
If we put the provider in `/usr/local/lib/summon` then users won't have to use summon's `-p` flag to pick it up. I think that's a better experience.

Reply:
Perfect. Anything we can do to clean up the summon call for a service manager would be great.